acme/acme/standalone.py

```diff
@@ -189,7 +190,7 @@ class HTTP01RequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
 
     def __init__(self, *args, **kwargs):
         self.simple_http_resources = kwargs.pop("simple_http_resources", set())
-        socketserver.BaseRequestHandler.__init__(self, *args, **kwargs)
+        BaseHTTPServer.BaseHTTPRequestHandler.__init__(self, *args, **kwargs)
```

Comment:
Somebody (possibly me) should double-check this

Reply:
like, how was this working?

Reply:
Terrifyingly, tests are passing.

Reply:
Ha! Ok so here's what's up: after several levels of indirection, `BaseHTTPServer.BaseHTTPRequestHandler` inherits from `socketserver.BaseRequestHandler`. This works because a) it's python so the init method isn't creating anything, just setting up some initial state and b) none of those levels of indirection define an `__init__` method, so the same `__init__` gets called that would have been called anyway.